doc/format-run.md

```diff
@@ -187,11 +203,10 @@ For this important special case, we can do significantly better than storing a
 whole key per page: we can typically store just 8 bytes (64bits) per page. This
 is a factor of 4 saving for 32 byte keys.
 
-Just as the bloom filter, the compact index file format starts with a 32 bit
-format identifier / format version, which determines the format and endianness
-of the rest of the file. This field is padded to 64 bits.
+The type of compact indexes is denoted by the number 0. The type–version
+indicater in a serialised compact index is padded to 64 bits.
```

Comment:
indicator

Reply:
Oh, no! This mistake again. Fixed 7374994e5e945729c5dfb45d79709e9f51674bcb.